npm/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "pizza",
+  "version": "0.0.0",
+  "description": "A command line utility for insights, metrics, and all things OpenSauced",
+  "repository": "https://github.com/open-sauced/pizza-cli",
+  "license": "MIT",
+  "bin": "./bin/runner.js",
+  "scripts": {
+    "install": "node install.js"
+  }
+}
```

Comment:
nit: Missing newline at end of file.

Reply:
https://github.com/open-sauced/pizza-cli/pull/31/commits/e51196e96f9a1bdac08190ed570cb5b62d523836